libp2p/src/main/kotlin/io/libp2p/pubsub/gossip/GossipParams.kt

```diff
@@ -231,7 +231,24 @@ data class GossipParams(
      * callback to notify outer system to which peers Gossip wants to be connected
      * The second parameter is a signed peer record: https://github.com/libp2p/specs/pull/217
      */
-    val connectCallback: (PeerId, ByteArray) -> Unit = { _: PeerId, _: ByteArray -> }
+    val connectCallback: (PeerId, ByteArray) -> Unit = { _: PeerId, _: ByteArray -> },
+
+    /**
+     * [maxIDontWantMessageIds] is the maximum number of IDONTWANT message ids allowed per heartbeat per peer
+     */
+    val maxIDontWantMessageIds: Int = maxIHaveLength * maxIHaveMessages,
+
+    /**
+     * [iDontWantMinMessageSizeThreshold] controls the minimum size (in bytes) that an incoming message needs to be so that an IDONTWANT message is sent to mesh peers.
+     * The default is 16 KB.
+     */
+    val iDontWantMinMessageSizeThreshold: Int = 16000,
```

Comment:
I know it doesn't matter but my mind works with 16384 :)

Reply:
Haha, was going to leave the same note, but strictly technically speaking 16 Kilobytes = 16000 bytes 😆

Reply:
yeah `KB` is stated not `KiB`, so it is consistent :-)